### DISCUSSION TITLE
GitLab Bot Security - now supports Group Access Tokens & API

### DISCUSSION THREAD

**Discussion by tennox** [renovatebot/renovate#15643] [Suggest an Idea]
Reading the recent changes in GitLab, I remembered the docs entry on ["GitLab Bot Security"](https://docs.renovatebot.com/gitlab-bot-security/) - and I think the situation might have changed now, as:
- Group Access Tokens are available and API-provisionable
- Personal Access Tokens are [API-provisionable](https://docs.gitlab.com/ee/api/project_access_tokens.html)
- as far as I understand, on self-hosted they are also all available in free tier (on gitlab.com I think you need a paid tier)

Is this sufficient to revisit the situation?

**Comment by viceice** [comment]
nope

blocked by https://gitlab.com/gitlab-org/gitlab/-/issues/346298

**Comment by rarkins** [comment]
We were hoping they'd fix the root cause/risk, but perhaps that's proven too hard. Both PATs and GATs are an inferior user experience to having an app or main "bot account", but we'll consider it if there's still no movement on the original security problem

**Reply by viceice** [reply]
There is slow movement, currently targeted to 15.4

https://gitlab.com/gitlab-org/gitlab/-/issues/346298

**Reply by tennox** [reply]
Thanks for the update & the efforts - I submitted a PR to add an update to the docs:
https://github.com/renovatebot/renovate/pull/15650

**Reply by tennox** [reply]
Kind of sad they don't see this use-case as important... I wonder if it conflicts with some design or business goal or it's just other things that are important...
But especially with the competition of GitHub, which makes this kind of situation really easy (I suppose) it's sad to see :/
